src/pages/task/taskTabs/buildBaronAndAnnotations/BBFileTicket.tsx

```diff
@@ -14,8 +14,8 @@ import { ButtonWrapper } from "./BBComponents";
 interface BBFileTicketProps {
   taskId: string;
   execution: number;
-  setCreatedTicketsCount: React.Dispatch<React.SetStateAction<number>>;
-  createdTicketsCount: number;
+  setCreatedTicketsCount?: React.Dispatch<React.SetStateAction<number>>;
+  createdTicketsCount?: number;
 }
 
 export const BBFileTicket: React.FC<BBFileTicketProps> = ({
```

Comment:
Is this component necessary it doesn't do anything but wrap  `<FileTicket />` ? We should either remove it or remove `<FileTicket />` and move the logic over.

Reply:
Ah yes, it probably evolved to a point where it is no longer necessary. Removing it.